### PR TITLE
Simplify javascript on backoffice pages

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Almost all components are globally initialized, we can just use them instead of bundling them again and again.
| Type?             | refacto
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Green tests should reveal an issue. Even a green asset build is a good indicator that everything is OK.
| UI Tests          | https://github.com/Hlavtox/ga.tests.ui.pr/actions/runs/10862142706
| Fixed issue or discussion?     |
| Related PRs       | 
| Sponsor company   | 

### Changes
- Changed all `grid.addExtension(new ExtensionName());` to `grid.addExtension(new window.prestashop.component.GridExtensions.ExtensionName());` and removed `import ExtensionName from '@components/grid/extension/path...';`
- Extensions `SubmitBulkExtension`, `SubmitGridExtension`, `ReloadListActionExtension` were changed to use the standard name.
- Use `ChoiceTable`, `TranslatableInput`, `ChoiceTree`, `CountryStateSelectionToggler`, `CountryDniRequiredToggler`, `GeneratableInput`, `TextWithRecommendedLengthCounter`, `MultipleChoiceTable` from `window.prestashop.component`

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

I found some issues with the Pull Request description:

* The `How to test` shouldn&#039;t be empty. ([Read explanation](https://devdocs.prestashop-project.org/9/contribute/contribution-guidelines/pull-requests/#how-to-test))

Would you mind having a look at it? This will help us understand how interesting your contribution is, thank you very much!

(Note: this is an automated message, but answering it will reach a real human)

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

I found some issues with the Pull Request description:

* The `How to test` shouldn&#039;t be empty. ([Read explanation](https://devdocs.prestashop-project.org/9/contribute/contribution-guidelines/pull-requests/#how-to-test))

Would you mind having a look at it? This will help us understand how interesting your contribution is, thank you very much!

(Note: this is an automated message, but answering it will reach a real human)

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

I found some issues with the Pull Request description:

* The `How to test` shouldn&#039;t be empty. ([Read explanation](https://devdocs.prestashop-project.org/9/contribute/contribution-guidelines/pull-requests/#how-to-test))

Would you mind having a look at it? This will help us understand how interesting your contribution is, thank you very much!

(Note: this is an automated message, but answering it will reach a real human)

**Comment by Hlavtox** [comment]
Tests green ✅

**Review by jolelievre — APPROVED** [review]
Thanks @Hlavtox

**Review by matthieu-rolland — APPROVED** [review]
nice 👍

**Comment by matthieu-rolland** [comment]
All UI tests are green 👍

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
